pages/index.js

```diff
@@ -25,6 +29,19 @@ function Home(props) {
     }
   }, []);
 
+  var yearsArray = []
```

Comment:
Either you can defined it as a function or directly add it to the html. 
Add a new button just to add new year to this drop down and check if list updates or not

Reply:
> Add a new button just to add new year to this drop down and check if list updates or not
It does not